ideascube/mediacenter/forms.py

```diff
@@ -27,3 +30,30 @@ def save(self, commit=True):
         document.save()
         self.save_m2m()
         return document
+
+
+def CreateDocumentForm(path):
+    class NotFileDocumentForm(forms.ModelForm):
+
+        class Meta:
+            model = Document
+            exclude = ['original', 'preview']
+
+        original = forms.FilePathField(path=path)
+        preview = forms.FilePathField(path=path, required=False)
+
+        def save(self, commit=True):
```

Comment:
Hum... indeed.

But I not sure of what we must not do if commit==False.
Do not do the save calls ?